packages/account/src/Assets/ic-blurry-document.svg

```diff
@@ -0,0 +1,155 @@
+<svg width="139" height="162" viewBox="0 0 139 162" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
are these minified? hard to tell

Reply:
not really , we tried with different way. the images were getting cropped